monkey/tests/unit_tests/infection_monkey/master/test_exploiter.py

```diff
@@ -38,12 +38,12 @@ def exploiter_config():
     return {
         "options": {"dropper_path_linux": "/tmp/monkey"},
         "brute_force": [
-            {"name": "HadoopExploiter", "supported_os": ["windows"], "options": {"timeout": 10}},
-            {"name": "SSHExploiter", "supported_os": ["linux"], "options": {}},
-            {"name": "WmiExploiter", "supported_os": ["windows"], "options": {"timeout": 10}},
+            {"name": "MSSQLExploiter", "options": {"timeout": 10}},
```

Comment:
I don't think this causes any issues, but why the change? The commit message provides no explanation.

Reply:
Updated the commit message
452028f221034722bedb635d3afc80e3dae13f0b